### ISSUE TITLE
captcha

### ISSUE THREAD

**Issue by Aashu-12** [Mennny/pokemeow-bot#1]
can u plz do something by which this solves captcha too?

**Comment by Mennny** [comment]
Rly someone try my bot lol :). 
Yeah i can try but i don't know nothing about that.

**Comment by Aashu-12** [comment]
> Rly someone try my bot lol :).
> Yeah i can try but i don't know nothing about that.

Yeah but if u can plz solve the captcha problem it will be so very good
